man/logind.conf.xml

```diff
@@ -355,6 +355,19 @@
         <citerefentry><refentrytitle>systemd.time</refentrytitle><manvolnum>7</manvolnum></citerefentry>.
         </para></listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><varname>MaintenanceWindow=</varname></term>
+
+        <listitem>
+          <para>
+            Specifies a default calendar event for scheduled shutdowns. So when using e.g. the command
+            <command>shutdown -r</command> to reboot the system without specifying a timeout, logind would
+            use the configured calendar event instead. For details about the syntax of calendar events, see
```

Comment:
I am strictly against extending sysv's "shutdown" command, it's not owned by us. Instead, extend the `--when=` switch to systemctl, to take a new explicit value `maintainance` or so

Reply:
Too late, already sneaked in #21680 in your absence :) Obviously doing something completely incompatible would be weird. It's not the case here though. It's just doing the right thing if no explicit time was specified.

I missed the addition of the --when parameter. From a quick look it uses 0 and inf as magic values and seems clumsy to use. `systemctl reboot --when=show`, really? :-)

Anyway I guess the design decision wrt maintenance window would be whether commands that trigger shutdown or reboot should automatically honor the maintenance window.

Reply:
i must admit that it classic shutdown is invoked without a time specification then it might be OK to pick the maintainance window instead of 1min. i.e. after all you are not really extending the syntax then, just tweaking the defaults under some condition.

so maybe i am OK with this after all

Reply:
but still the "--when=" thing in systemctl should really the primary iface, hence do --when=maintainance or so to it at the same time please

Reply:
added --when=auto